scripts/notify-suppliers-of-new-questions-answers.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+"""
+Send email notifications to supplier users when a question has been answered
+
+Usage: notify-suppliers-of-new-questions-answers.py <stage> <api_token> <mandrill_api_key> [options]
+
+Options:
+    --dry-run                               List notifications that would be sent without sending the emails
+    --exclude-supplier-ids=EXCLUDE-SUPPLIERS   Comma separated list of suppliers IDs to be excluded. This is incase the
```

Comment:
We actually planned to use notify for this script and we decided to move to mandrill midway through because of how email to each supplier group of addresses would have different number of briefs. There were also big problems with formatting. @idavidmcdonald and @pcraig3 can tell you more about this.

As to input of the excluded supplier ids into command line you have a point there. I din't know the technicalities around passing in a file though.